prow/crier/reporters/gerrit/reporter.go

```diff
@@ -407,31 +418,39 @@ func (c *Client) Report(ctx context.Context, logger *logrus.Entry, pj *v1.ProwJo
 	if err := c.gc.SetReview(gerritInstance, gerritID, gerritRevision, message, reviewLabels); err != nil {
 		logger.WithError(err).WithField("gerrit_id", gerritID).WithField("label", reportLabel).Info("Failed to set review.")
 
-		if reportLabel == "" {
-			return nil, nil, err
+		// It could be that the commit is deleted by the time we want to report.
+		// Swollow the error if this is the case.
+		exist, existErr := c.gc.ChangeExist(gerritInstance, gerritID)
```

Comment:
We should probably catch if existErr!=nil

Reply:
I don' t think existErr is super useful, the outcome is the same as SetReview error. Added a log line for existErr